ProjectTemplates/VisualStudio2015/WindowsUniversal10/WindowsUniversal10Xaml.vstemplate

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <VSTemplate Version="3.0.0" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" Type="Project">
   <TemplateData>
-    <Name>MonoGame Windows 10 Universal Project</Name>
-    <Description>A MonoGame game project for Windows 10 UWP.</Description>
+    <Name>MonoGame Windows 10 Universal (XAML) Project</Name>
```

Comment:
I would say change this to:

`MonoGame Windows 10 Universal XAML Project`


@KonajuGames @dellis1972 ?

Reply:
So as to distinguish between the two templates, one being XAML and the other being a core / D3D application.
Just saying Win 10 Universal alone isn't enough to describe what it is